game_for_class.py

```diff
@@ -16,26 +23,54 @@ def __init__(self, name, hp, attack, defense, speed, team):
         self.team = team
 
     def is_alive(self) -> bool:
-        """returns true if the character is alive"""
-        return self.hp >= 1
+        """Returns true if the character is alive"""
```

Comment:
less than or equal to 0 meaning it'll return the character alive at 0 hp

Reply:
`self.hp > 0` means `True` if greater than 0. It will return `False` if hp is 0 or less